app/assets/stylesheets/views/_dit-landing-page.scss

```diff
@@ -0,0 +1,86 @@
+$dark-red: #C21D26;
+$dark-grey: #282828;
+
+.dit-landing-page__header {
+  padding-bottom: govuk-spacing(6);
+  padding-top: govuk-spacing(3);
+  margin-bottom: govuk-spacing(6);
+  background-color: $dark-red;
+  color: govuk-colour("white");
+}
+
+.dit-landing-page__header--with-bg-image {
+  padding-bottom: govuk-spacing(4);
+  padding-top: govuk-spacing(3);
+  background-color: $dark-red;
+  border-top: govuk-spacing(2) solid $dark-red;
+
+  @include govuk-media-query($from: $header-breakpoint) {
+    padding-bottom: govuk-spacing(4);
+    padding-top: govuk-spacing(8);
+    background-image: image-url('DIT-banner@2x.png');
+    background-size: cover;
+    background-position: top;
+    min-height: 500px;
+  }
+
+  // HIDE BACKGROUND IMAGE ON IE
+
+  // IE 6, 7, 8
+  @media \0screen\,screen\9 {
+    background-image: none;
+  }
+
+  // IE 9
+  @media screen and (min-width:0\0) and (min-resolution: .001dpcm) {
+    background-image: none;
+  }
+}
+
+.dit-landing-page__header-text-block {
+  @include govuk-media-query($from: $header-breakpoint) {
+    padding-bottom: 80px;
+    padding-top: govuk-spacing(5);
+
+    background: $red;
+    background: -moz-linear-gradient(90deg, $dark-red 66.66%, $transparent 66.66%);
+    background: -webkit-linear-gradient(90deg, $dark-red 66.66%, $transparent 66.66%);
+    background: linear-gradient(90deg, $dark-red 66.66%, $transparent 66.66%);
+  }
+}
+
+.dit-landing-page__intro {
+  p {
+    @include govuk-font($size: 24);
+    color: govuk-colour("white");
+  }
+
+  a:link,
+  a:visited {
+    font-weight: bold;
+    color: govuk-colour("white");
+  }
+
+  a:focus {
+    color: govuk-colour("black");
+  }
+}
+
+.dit-landing-page__page_title {
+  @include govuk-font($size: 48, $weight: bold);
+  color: govuk-colour("white");
+  margin-top: govuk-spacing(7);
+  margin-bottom: govuk-spacing(4);
+}
+
+.dit-landing-page__section--dark-grey {
+  @include govuk-responsive-padding(8, "top");
+  @include govuk-responsive-padding(8, "bottom");
+  background-color: $dark-grey;
+  margin-top: 0;
+
+  h2,
```

Comment:
non-blocker: we generally tend to avoid styling elements directly, but I do get these are coming from govspeak so it's quicker to do it this way.